package.json

```diff
@@ -39,12 +39,8 @@
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s -u"
   },
   "dependencies": {
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.4.4",
-    "@babel/plugin-proposal-optional-chaining": "^7.6.0",
-    "@babel/plugin-transform-typescript": "~7.7.0",
```

Comment:
1. To make this work, will we need ember-cli-babel to introspect against not only this but the `typescript` version installed to make sure things are set up correctly?
2. Don't *we* need these internally for our *own* compilation pipeline? Or is the idea that our dependency on `ember-cli-babel` will handle it for us as well? (I'm trying to think through the circularity here.)

Reply:
1. That was my understanding and what I did in https://github.com/babel/ember-cli-babel/pull/314
2. We really don't have our *own* compilation pipeline (well, we do, but it doesn't use Babel).

Reply:
1. For the same reason we [couldn't do it here](https://github.com/typed-ember/ember-cli-typescript/pull/936), we unfortunately can't conditionalize plugin inclusion on our parent's TS version in ember-cli-babel. Tl;dr: for anything other than the root app, we don't know what version of TypeScript the package was running against; at best we just have a semver specifier.

2. What @jamescdavis said 🙂 — our build files are transpiled via `tsc`, and all our runtime test code will be handled by ember-cli-babel once it's updated.

Reply:
DOH. Thank you both for patiently clarifying for me. (Let’s just write this off as a symptom of my just having spent 3½ months upgrading a massive app which had a ton of private router API usage from 3.4 to 3.8. :sob:)

Reply:
1. I was pretty sure that was the case, but I figured I'd go ahead and add the conditionality and we could remove it if it needed.